src/main/scala/org/broadinstitute/dsde/snoop/ws/CORSDirectives.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsde.snoop.ws
+
+import java.io.File
+
+import com.typesafe.config.ConfigFactory
+import spray.http.HttpHeaders._
+import spray.http.HttpMethods._
+import spray.http._
+import spray.routing._
+/**
+ * https://gist.github.com/joseraya/176821d856b43b1cfe19
+ * https://github.com/giftig/mediaman/blob/22b95a807f6e7bb64d695583f4b856588c223fc1/src/main/scala/com/programmingcentre/utils/utils/CorsSupport.scala
+ */
+case class `Access-Control-Allow-Credentials`(allowed: Boolean) extends HttpHeader {
+  def name = "Access-Control-Allow-Credentials"
+  def lowercaseName = "access-control-allow-credentials"
+  def value = if(allowed) "true" else "false"
+
+  override def render[R <: Rendering](r: R): r.type = r ~~ allowed.toString
+}
+
+trait CORSDirectives {
```

Comment:
I have a very strong preference toward not capitalizing acronyms in names for classes and such. I know there's a mix in our combined DSDE code bases but I'm choosing this moment to put up a fuss about it.